src/cmd/server/main.go

```diff
@@ -56,7 +56,7 @@ func getConf() {
 	}
 
 	if conf.SFTPGoEnabled {
-		err = dbi.SFTPConnect(conf.SFTPGoAPIURL, conf.SFTPGoAdminLogin, conf.SFTPGoAdminPassword)
+		err = dbi.SFTPConnect(conf.SFTPGoAPIURL, conf.SFTPGoAdminLogin, conf.SFTPGoAdminAPIKey)
```

Comment:
Get rid of the `SFTPGoAdminLogin` from anywhere it's used - it shouldn't be passed around anymore, or even configured by users.

Reply:
I believe my thinking at the time was that we still needed it because internally there is still a username that is associated with any API key and multiple users could be issued an API key, but upon revisiting I realize that the API key HTTP header string contains a string for the user as well so you are right this is no longer needed. :+1: